pkg/controllers/customcluster_controller.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	clusterv1alpha1 "kurator.dev/kurator/pkg/apis/cluster/v1alpha1"
+)
+
+// CustomClusterReconciler reconciles a CustomCluster object
+type CustomClusterReconciler struct {
```

Comment:
s/CustomClusterReconciler/CustomClusterController

Reply:
Does it mean that the path should be changed to pkg/controllers/custom-cluster-reconciler/customcluster_controller.go
?